src/components/StoreInfo/StoreLocation.tsx

```diff
@@ -17,15 +17,56 @@ const StoreLocation: FC<Props> = ({ locations }) => {
                   className={styles.wrap__singleLocation}
                   key={location.seller_id}
                 >
-                  <address>
-                    <p className={styles.address}>
-                      {location.address1 + (location.address2 || '')}
+                  <address
+                    itemProp="address"
+                    itemType="https://schema.org/PostalAddress"
+                  >
+                    <p itemProp="streetAddress" className={styles.address}>
+                      {location.address1}
+                      {location.address2 ? <br /> : ''}
+                      {location.address2 ? location.address2 : ''}
                     </p>
                     <p className={styles.address}>
-                      {location.city}, {location.state} {location.zip_code}
+                      <span itemProp="addressLocality">{location.city}</span>,{' '}
+                      <span itemProp="addressRegion">{location.state}</span>{' '}
+                      <span itemProp="postalCode">{location.zip_code}</span>
                     </p>
                   </address>
-                  <p className={styles.address}>{location.phone_number}</p>
+                  {location.address1 && (
+                    <span>
+                      <a
+                        href={
+                          'https://www.google.com/maps/place/' +
+                          (
+                            (location.address1 + ' ' || '') +
+                            (location.city + ' ' || '') +
+                            (location.state + ' ' || '') +
+                            (location.zip_code || '')
+                          ).replaceAll(' ', '+')
+                        }
+                        target="_blank"
```

Comment:
@carolinexsun it looks like the error is showing. could you add this attribute to the target tag? 

Failed to compile.
```
./src/components/StoreInfo/StoreLocation.tsx
  Line 47:25:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank

error Command failed with exit code 1.

````

Reply:
Yup, can re-add. It seemed like an optional value to block tracking of referral data to external links, but apparently might be less optional than I thought.